src/python/pants/notes/1.15.x.rst

```diff
@@ -3,6 +3,16 @@
 This document describes releases leading up to the ``1.15.x`` ``stable`` series.
 
 
+1.15.0 (4/15/2019)
+------------------
+
+The first stable release of the `1.15.x` series, with no changes since `rc3`!
+
+This is the first Pants version to support being run under either Python 3.6 or 2.7. If you've been
+using the `./pants` script we provide prior to this release, update the script by
```

Comment:
Thought of a better way to word this:

```
If you've been using our provided `./pants` script prior to this release,
```

The original was confusing to read.